### PR TITLE
Portable Multifunction Device

### PR DESCRIPTION
**Pull request by MangoTornado**
## Describe your project in 1-3 sentences

I am building a portable multifunction device, that can assist in general applications as well as penetration testing the smart/not-smart devices around my home. I want to start learning how to solder and create prototyping boards for testing air quality, measuring distances, sniffing packets, and etc.

I will be able to cover the extra cost including shipping and etc.

## Hack Club Slack handle

linuxpenguin

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: MangoTornado.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Flipper Zero    | [https://shop.flipperzero.one/](https://shop.flipperzero.one/)         | $169.00  |
+| Adafruit BME688 | [https://www.adafruit.com/product/5046](https://www.adafruit.com/product/5046)  | $19.95 |
+| HC-SR04 | [https://www.adafruit.com/product/3942](https://www.adafruit.com/product/3942)  | $3.95 |
+| PINECIL – Smart Mini Portable Soldering Iron | [https://www.amazon.com/dp/B096X6SG13](https://www.amazon.com/dp/B096X6SG13)  | $39.99 |
+| Hakko 633-01 633 Soldering Iron Holder | [https://www.amazon.com/dp/B000TGNWCS](https://www.amazon.com/dp/B000TGNWCS)  | $22.20 |
```
Comment (exu3) [inline MangoTornado.md:24]: If you don't already own any, I might recommend getting other soldering tools (like a solder sucker, solder wick, flux). You may find them helpful but not necessary.

File: MangoTornado.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+| Prototyping Boards for Flipper Zero | [https://shop.flipperzero.one/products/proto-boards](https://shop.flipperzero.one/products/proto-boards)  | $10.00 |
+| Dupont Wire | [https://www.amazon.com/dp/B01EV70C78/](https://www.amazon.com/dp/B01EV70C78/)  | $6.98 |
```
Comment (exu3) [inline MangoTornado.md:26]: Will you need any passives (resistors, capacitors, diodes, etc.) for your circuit?

**Review by exu3 — CHANGES_REQUESTED** [review]
@MangoTornado neat idea! I like how your `Plan` lists out the various components in your budget and how you plan to use them.

I left a few comments on your budget, but other than that, I think your project is a go! Once you address those comments, feel free to request another review.

File: MangoTornado.md (hunk 1 above)
Comment (exu3) [reply MangoTornado.md:24]: Would you also need solder?

Comment (exu3) [reply MangoTornado.md:24]: Would you also need solder wire?

Comment (MangoTornado) [reply MangoTornado.md:24]: Solder sucker is a good idea; I hadn't considered messing up :p

File: MangoTornado.md (hunk 2 above)
Comment (MangoTornado) [reply MangoTornado.md:26]: The ultrasonic chip HC-SR04 comes with 2 resistors, and I don't think any are necessary for the BME

**Review by exu3 — APPROVED** [review]
Nice work! The @hackclub/bank-ops team will follow up with next steps for receiving the grant & getting onto Hack Club Bank. 

Looking forward to seeing what you ship.

**Review by arianna-m — APPROVED** [review]
Hi @MangoTornado! Wow, this looks so cool! I like how detailed your plan is and how you're interested in learning how to solder. Next steps for us would be to move communications over to Slack so we can move forward! We can schedule a Zoom call, so I can explain how you get the grant and share with you the fiscal sponsorship contract which you'll have to sign with a parent/guardian. Looking forward to working together! :)

**Pull request merged**